userdocs/src/usage/cloudwatch-cluster-logging.md

```diff
@@ -56,25 +56,33 @@ eksctl utils update-cluster-logging --disable-types all
 
 ## `ClusterConfig` Examples
 
-There 5 types of logs that you may wish to enable (see [EKS documentation][eksdocs] for more details):
+In an EKS cluster, the "enableTypes" field under "clusterLogging" can take a list of possible values to enable the different types of logs for the control plane components.
 
-- `api`
-- `audit`
-- `authenticator`
-- `controllerManager`
-- `scheduler`
+### Values
+The following are the possible values:
```

Comment:
```suggestion
The following are the possible values:
```
Remove the Values heading, it looks odd. Add the values as part of ## `ClusterConfig` Examples for better readability

![Screenshot 2023-03-09 at 11 58 53](https://user-images.githubusercontent.com/10995114/224017432-cd58f7c3-8107-4ec1-bc9b-0ca60abecc5f.png)